apps/web/src/services/dashboards/widgets/_helpers/widget-table-helper.ts

```diff
@@ -37,7 +37,7 @@ export const getWidgetTableDateFields = (
     let count = 0;
     while (now.isSameOrBefore(end, timeUnit)) {
         dateFields.push({
-            name: `${fieldsKey}_sum.${count}.value`,
+            name: `${fieldsKey}.${count}.value`,
```

Comment:
description: The return value of the API used by the asset widget does not include values in such a format. (_sum)